src/main/java/kirby/Kirby.java

```diff
@@ -0,0 +1,72 @@
+package kirby;
+
+import kirby.commands.Command;
+import kirby.exceptions.KirbyInvalidCommandException;
+import kirby.exceptions.KirbyMissingArgumentException;
+import java.io.*;
+
+/**
+ * Kirby class implements the main method.
+ */
+public class Kirby {
+    private Storage storage;
+    private TaskList tasks;
+    private Ui ui;
+
+    /**
+     * Constructor for the class Kirby.
+     *
+     * @param filePath name of the file path as a string.
```

Comment:
Not too sure myself but I believe the first letter of of the `param`, `return` and `throws` statements should be captitalised
```suggestion
     * @param filePath Name of the file path as a string.
```

Reply:
You're right thanks!